src/rootfs.rs

```diff
@@ -940,4 +938,139 @@ mod tests {
             .get_symlink_args();
         assert_eq!(want, got)
     }
+
+    #[test]
+    #[serial]
+    fn test_bind_dev() {
+        let rootfs = RootFS::new();
+        assert!(rootfs
+            .bind_dev(
+                Path::new("/tmp"),
+                &LinuxDeviceBuilder::default()
+                    .path(PathBuf::from("/null"))
+                    .build()
+                    .unwrap(),
+            )
+            .is_ok());
+
+        let want = MountArgs {
+            source: Some(PathBuf::from("/null")),
+            target: PathBuf::from("/tmp/null"),
+            fstype: Some("bind".to_string()),
+            flags: MsFlags::MS_BIND,
+            data: None,
+        };
+        let got = &rootfs
+            .syscall
+            .as_any()
+            .downcast_ref::<TestHelperSyscall>()
+            .unwrap()
+            .get_mount_args()[0];
+        assert_eq!(want, *got);
+
+        remove_file("/tmp/null").unwrap();
```

Comment:
I should have pointed this out in an earlier test, but how about using this?
https://github.com/containers/youki/blob/c4264a1096f417f2f5348c56dba375bea97af22b/src/utils.rs#L221

Reply:
sure will address this